Resources/Prototypes/_NF/Entities/Structures/Machines/lathe.yml

```diff
@@ -385,3 +388,242 @@
+    - NetworkConfigurator
+    - HandheldGPSBasic
+    - TRayScanner
+    - AirTank
+    - GasAnalyzer
+    - AppraisalTool
+    - HandLabeler
+    - LightTube
+    - LedLightTube
+    - SodiumLightTube
+    - ExteriorLightTube
+    - LightBulb
+    - LedLightBulb
+    - DoorElectronics
+    - StationMapElectronics
+    - CellRechargerCircuitboard
+    - BorgChargerCircuitboard
+    - WeaponCapacitorRechargerCircuitboard
+    - RadioHandheld
+    - FlashlightSeclite
+    - Floodlight
+    - FlareLathe
+    - GlowstickBlue
+    - GlowstickBase
+    - GlowstickPurple
+    - GlowstickRedLathe
+    - GlowstickYellow
+    - Zipties
+    - TargetClown
+    - TargetHuman
+    - CableStack
+    - PowerCellSmall
+    - PowerCellMedium
+    - SignalTimerElectronics
+    # Weapons
+    - CombatKnife
+    - WeaponLaserSvalinn
+    - WeaponLaserPistolNF
+    - WeaponRevolverArgenti
+    - WeaponPistolMk58NF
+    - WeaponShotgunDoubleBarreled
+    - WeaponRifleNovaliteC1
+    - WeaponRifleGestio
+    # Ammo boxes
+    - BoxLethalshot
+    - BoxShotgunFlare
+    - BoxShotgunSlug
+    - BoxShellTranquilizer
+    - MagazineBoxLightRifle
+    - MagazineBoxMagnum
+    - MagazineBoxPistol
+    - MagazineBoxRifle
+    # Mags / Speedloaders / Clips
+    - MagazineNovaliteC1Empty
+    - MagazineNovaliteC1
+    - MagazineLightRifleLowCapacityEmpty
+    - MagazineLightRifleLowCapacity
+    - MagazineLightRifle
+    - MagazineLightRifleEmpty
+    - MagazineRifle
+    - MagazineRifleEmpty
+    - MagazinePistol
+    - MagazinePistolEmpty
+    - SpeedLoaderMagnum
+    - SpeedLoaderMagnumEmpty
+    - SpeedLoaderRifleHeavy
+    - SpeedLoaderRifleHeavyEmpty
+    - MagazineShotgun
+    - MagazineShotgunEmpty
+    - MagazineShotgunSlug
+    - 10roundmagazine #frontier by VividPups
+    - 10roundmagazineempty #frontier by VividPups
+    dynamicRecipes:
+    # Armor
+    - TelescopicShield
+    - ClothingOuterArmorBulletproofNF
+    - ClothingOuterCoatBHTrench
+    # EVA / Hardsuits
+    - ClothingOuterHardsuitCap
+    - ClothingOuterHardsuitMercenary
+    - ClothingOuterHardsuitPrivateSecurity
+    - ClothingOuterHardsuitScaf
+    # Utilities
+    - ClothingEyesHudBountyHunter
+    - ClothingEyesPunkInfoShades
+    - PinpointerUniversalNF
+    - Handcuffs
+    - JetpackVoid
+    - SignalTrigger
+    - PortableRecharger
+    - TimerTrigger
+    - VoiceTrigger
+    - PowerCellMicroreactor
+    - PowerCellMedium
+    - PowerCellHigh
+    - PowerCellMicroreactor
+    - TurboItemRechargerCircuitboard
+    - ClothingBackpackHolding
+    - ClothingBackpackSatchelHolding
+    - ClothingBackpackDuffelHolding
+    - ClothingBackpackMessengerHolding
+    - WelderExperimental
+    - JawsOfLife
+    - PowerDrill
+    # - HoloprojectorSecurity
+    # Weapons
+    - BaseBallBatNF
+    - KatanaNF
+    - KukriKnifeNF
+    - MacheteNF
+    - Stunbaton
+    - BolaNF
+    - ExplosivePayload
+    - FlashPayload
+    - ChemicalPayload
+    - WeaponShotgunKammererNF
+    # - WeaponAdvancedLaser
```

Comment:
The advanced laser pistol is (currently) civilian tech. Why not make it available?

Reply:
Fair enough, I'll uncomment it